packages/@webex/internal-plugin-metrics/src/metrics.types.ts

```diff
@@ -110,6 +110,8 @@ export type MetricEventVerb =
   | 'warn'
   | 'exit';
 
+export type MetricEventJoinFlowVersion = 'Other' | 'NewFTE';
```

Comment:
question: What does NewFTE indicate? I was unable to get this information from the media SPARK jira too.

Reply:
New First Time Experience.  It just means the consolidated interstitial flow (where we skip straight to the interstitial page for guest users and allow them to enter their display name there).

Reply:
![image](https://github.com/user-attachments/assets/462d08ce-d858-4b64-b919-b3f636bd3964)